db/migrate/20221121193912_add_comment_to_review_policy_classes.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+class AddCommentToReviewPolicyClasses < ActiveRecord::Migration[6.1]
+  def change
+    add_column :review_policy_classes, :comment, :string
+  end
```

Comment:
Doesn't matter in the least, but could these three migrations be merged into one? Or is there a specific reason for keeping them separate.

Reply:
The commits demonstrate the order I did things in including the minimum migration changes. I was going to squash the commit but, since you've mentioned it, I can also squash the migrations. 👍